examples/gpio-demo/slightfile.toml

```diff
@@ -0,0 +1,10 @@
+specversion = "0.2"
+
+[[capability]]
+resource = "gpio.raspberry_pi"
+name = "my-gpio"
+    [capability.configs]
+    push_down_button = "17/input/pulldown"
+    led = "5/output"
+    pwm_control_button = "18/input/pullup"
+    pwm_led = "6/pwm_output"
```

Comment:
I see that you're defining pin config. (i.e., pull-up/pull-down) directly in the string vars here. Have you considered baking this into some sort of structure within the interface?